tools/notification-configuration/identity-resolver/Program.cs

```diff
@@ -34,7 +34,9 @@ string targetvar
         {
 
 #pragma warning disable CS0618 // Type or member is obsolete
-            using (var loggerFactory = new LoggerFactory().AddConsole(includeScopes: true))
+            using (var loggerFactory = LoggerFactory.Create(builder => {
```

Comment:
@scbedd -- Upgraded logging here because `MemoryCache` moved a transitive dependnecy up. Tested that this program runs and outputs as expected.

Reply:
I don't mind but do we still need this update now that you aren't using MemoryCache?

Reply:
We don't specifically need it but since I did this work I think it's better to keep the upgraded version than to revert back. I think we eventually may move this version forward anyway so this saves us a small bit of headache in the future.